src/util/octal_utils.e

```diff
@@ -83,7 +113,7 @@ feature -- Parsing
 			leading_zeros: INTEGER
 		do
 			digit_weight := 1
---			Result := 0
+				-- Result is set by default to 0
```

Comment:
when you have commented code .. either remove the line, or comment why the code is commented